### ISSUE TITLE
Request param `filter[limit]` ignored

### ISSUE THREAD

**Issue by reneras** [prototype-berlin/loopback-paginator#3]
**Issue**
Request param `filter[limit]` ignored by loopback-paginator mixin

**Scenario**
There are cases when you want to provide a `filter[limit]` in a request to loopback. For example to have an UI with a dynamic page limit, where the user can select how many records to show per page.

**Comment by mahnuh** [comment]
@reneras this feature is already on the road map, but I do not know when I will have time to work on it.

**Comment by mahnuh** [comment]
Oh, just saw you PR @reneras, will take a look at it later!

**Comment by mahnuh** [comment]
Hi @reneras,

I just published v1.0.0 of loopback-paginator which allows LoopBack limit filters! 🎉
Hope you don't mind that I did not use your PR, but I already had some other stuff in mind which I've now implemented. Make sure to check out the readme, especially the part with the new [limit and maxLimit options](https://github.com/prototype-berlin/loopback-paginator#limit-and-maxlimit).
I hope the changes work for you, just let me know if you run into any issues!

Manuel
